client/landing/stepper/declarative-flow/internals/steps-repository/site-migration-instructions/steps/use-steps.tsx

```diff
@@ -121,52 +124,60 @@ const useStepsData = ( { fromUrl }: StepsDataOptions ): StepsData => {
 		},
 		{
 			title: translate( 'Add your migration key' ),
-			content: (
-				<>
-					<p>
-						{ translate(
-							'Copy the key below. Head to the Migrate Guru settings on your source site, and paste it into the {{strong}}%(migrationKeyLabel)s{{/strong}} field.',
-							{
+			content:
+				'' === migrationKey ? (
+					<>
+						<p>{ translate( 'The key will be available here when your new site is ready.' ) }</p>
```

Comment:
This is a little different than the design. See the discussion here: p9Jlb4-c8w-p2#comment-12208